core/arch/arm/kernel/spmc_sp_handler.c

```diff
@@ -57,9 +57,63 @@ static TEE_Result ffa_get_dst(struct thread_smc_args *args,
 	return FFA_OK;
 }
 
+static struct sp_mem_receiver *get_sp_mem_receiver(struct sp_session *s,
```

Comment:
Functions with `get` in the name sometimes has side effects, like increasing a reference counter or such.
We are not enforcing anything specific with these `get` functions but they do stand out a bit.
How about calling this function `find_sp_mem_receiver()` instead to make it clear that it's just a simple lookup function without side effects?

Reply:
Done